Sources/Java2SwiftLib/JavaTranslator.swift

```diff
@@ -265,7 +274,7 @@ extension JavaTranslator {
     )
 
     if !enumConstants.isEmpty {
-      let enumName = "\(swiftTypeName)Cases"
+      let enumName = "\(swiftTypeName.splitSwiftTypeName().name)Cases"
```

Comment:
This can be `swiftInnermostTypeName`, right?
```suggestion
      let enumName = "\(swiftInnermostTypeName)Cases"
```

Reply:
yep missed that